bundles/org.openhab.binding.wiz/README.md

```diff
@@ -0,0 +1,144 @@
+| ipAddress         | text           | true      | The IP of the bulb                                                                                                                                                                                                                                                                            |         |
+| updateInterval    | integer        | false     | Update time interval in seconds to request the status of the bulb.                                                                                                                                                                                                                            | 60      |
+| useHeartBeats     | boolean        | false     | Whether to register for continuous 5s heart-beats                                                                                                                                                                                                                                             | false   |
+| reconnectInterval | integer        | false     | Interval in minutes between attempts to reconnect with a bulb that is no longer responding to status queries.  When the bulb first connects to the network, it should send out a firstBeat message allowing openHAB to immediately detect it.  This is only as a back-up to re-find the bulb. | 15      |
+
+Example Thing:
+
+```java
+Thing wiz:bulb:lamp "My Lamp" @ "Living Room" [ macAddress="accf23343cxx", ipAddress="192.168.0.xx" ]
+```
+
+## Channels
+
+The binding supports the following channels:
+
+| Channel ID      | Item Type          | Description                                           | Access |
+|-----------------|--------------------|-------------------------------------------------------|--------|
+| color           | Color              | State, intensity, and color of the LEDs               | R/W    |
+| temperature     | Dimmer             | Color temperature of the bulb                         | R/W    |
+| temperature-abs | Number:Temperature | Color temperature of the bulb in Kelvin               | R/W    |
+| brightness      | Dimmer             | The brightness of the bulb                            | R/W    |
+| state           | Switch             | Whether the bulb is on or off                         | R/W    |
+| light-mode      | Number             | Preset light mode name to run                         | R/W    |
+| speed           | Dimmer             | Speed of the color changes in dynamic light modes     | R/W    |
+| signal-strength | Number             | Quality of the bulb's WiFi connection                 | R      |
+| last-update     | Time               | The last time an an update was received from the bulb | R      |
+
```

Comment:
I think some fan channels are missing here, e.g. `mode` and `reverse`. Also some channels seem to have same name, but different meaning, so maybe needs to be provided per Thing type? For example `speed` is different for fan.